metrics/configs/flakes-experiment-config.yaml

```
@@ -55,22 +55,22 @@ query: |
           SELECT
             job,
             if(substr(job, 0, 3) = 'pr:', 'pull', 'ci') kind,  /* pull or ci */
-            ifnull(repo_commit, version) version, /* bootstrap git version, empty for ci  */
+            ifnull(repo_commit, version) version, /* git version, empty for ci  */
             if(substr(job, 0, 3) = 'pr:',
               regexp_extract(
                 repos,
                 r'[^,]+,\d+:([a-f0-9]+)"'
               ),
-              version
+              ifnull(repo_commit, version)
             ) commit,  /* repo commit for PR or version for CI */
             result,  /* SUCCESS if the build passed */
             test  /* repeated tuple of tests */
-          FROM `k8s-gubernator.build.week` as t
+          FROM `k8s-gubernator.build.staging` as t
           where
             datetime(started) > datetime_sub(current_datetime(), interval 7 DAY)
-            and version != 'unknown'
+            and (version != 'unknown' or repo_commit is not null)
             and (
-              (substr(job, 0, 3) = 'ci-' and version != 'unknown') or
+              (substr(job, 0, 3) = 'ci-' and (version != 'unknown' or repo_commit	is not null)) or
```

nit for followup (since this is also in the original and you're just following what's there): since we check `(version != 'unknown' or repo_commit is not null)` above, I don't think we need to do it again
```suggestion
              (substr(job, 0, 3) = 'ci-' or
```